moveit_core/collision_detection_fcl/include/moveit/collision_detection_fcl/collision_env_fcl.h

```diff
@@ -154,6 +154,9 @@ class CollisionEnvFCL : public CollisionEnv
   /** \brief Callback function executed for each change to the world environment */
   void notifyObjectChange(const ObjectConstPtr& obj, World::Action action);
 
+  /** \brief Check if enable_nearest_distance_points is available */
+  void checkFCLNearestDistCapability(const DistanceRequest& req) const;
```

Comment:
This is not added to #505

Reply:
this one also doesn't exist in https://github.com/ros-planning/moveit/pull/2682/files